core/src/utils/test/overlays/overlays.e2e.ts

```diff
@@ -104,6 +104,53 @@ configs({ modes: ['ios'], directions: ['ltr'] }).forEach(({ title, config }) =>
       const modals = page.locator('ion-modal');
       expect(await modals.count()).toEqual(0);
     });
+
+    test('should re-enable scrolling on the page when dismissed', async ({ page, skip }) => {
+      skip.browser('webkit', 'Mouse wheel is not supported in mobile WebKit');
+      await page.goto('/src/utils/test/overlays', config);
+
+      await page.click('#create-and-present');
+
+      const modal = page.locator('ion-modal');
+      await expect(modal).toHaveCount(1);
+
+      const contentEl = page.locator('#main-content ion-content');
+
+      await expect(
+        await contentEl.evaluate(async (el: HTMLIonContentElement) => {
+          const contentScrollEl = await el.getScrollElement();
+          return contentScrollEl.scrollTop;
+        })
+      ).toBe(0);
+
+      await page.mouse.wheel(0, 100);
```

Comment:
I'm not entirely happy with the way that I had to test scrolling inside Playwright here. The `page.mouse.wheel` API is unavailable in mobile webkit/Safari. However any other emulation of adjusting `scrollTop`, or using `scrollIntoViewWhenNeeded` does not actually perform a scroll, just manually adjusts the scroll position.